packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -116,7 +117,7 @@ export class AtomicResultList implements InitializableComponent {
 
   private registerDefaultResultTemplates() {
     this.resultTemplatesManager.registerTemplates({
-      content: defaultTemplate,
+      content: {innerHTML: defaultTemplate, usesSections: false},
```

Comment:
Why is a `usesSections` prop needed? Why not inspect the `innerHTML` to see if it is using sections?

Reply:
This information is easier to parse at the template level because at that point it already uses DOM elements and hasn't been rendered yet. Also, if in the future we want to make this an option that can be forced or we want to support an alternative to sections, it's easier to adjust at the template level.